src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TemporaryAbsence.kt

```diff
@@ -14,8 +15,10 @@ data class TemporaryAbsence(
   val lastName: String,
   @Schema(description = "Date of birth", example = "1971-02-01")
   val dateOfBirth: LocalDate,
+  @JsonAlias("offenderNo")
   @Schema(description = "Prison number", example = "A1234AA")
   val prisonNumber: String?,
-  @Schema(description = "Reason", example = "Hospital")
+  @JsonAlias("movementReason")
```

Comment:
possibly a bit non obvious whats going on here - that we're using alias to read a different field than we produce rather than allowing reading different fields into the same field

Reply:
think it's probably fine for now